tetrateci/test_1.7.sh

```diff
@@ -31,7 +31,7 @@ for package in $PACKAGES; do
   do
     echo "========================================================TRY $n========================================================"
     go test -count=1 -p 1 -test.v -tags=integ $package -timeout 30m --istio.test.select=-postsubmit,-flaky ${CLUSTERFLAGS} && break || echo "Test Failed: $package"
-    sudo rm -rf $(ls /tmp | grep istio)
+    sudo rm -rf -- $(ls /tmp | grep istio)
```

Comment:
What is the directory we are trying to remove here?

Reply:
it is in the format `/tmp/<path to the package with slashes replaces by hyphens>` like, `/tmp/-home-runner-istio-istio-tests-integration-pilot-`